src/olympia/files/utils.py

```diff
@@ -298,6 +298,8 @@ def apps(self):
                 _('GUID is required for Firefox 47 and below.')
             )
 
+        couldnt_find_version = False
```

Comment:
Well it has to be defined before the for-loop since I want it to be set whenever we find an app that does not exist.